packages/flutter/test/widgets/scroll_notification_test.dart

```diff
@@ -7,6 +7,64 @@ import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  testWidgets('ScrollMetricsNotification test', (WidgetTester tester) async {
+    final List<LayoutChangedNotification> events = <LayoutChangedNotification>[];
+    Widget buildFrame(double height) {
+      return NotificationListener<LayoutChangedNotification>(
+        onNotification: (LayoutChangedNotification value) {
+          events.add(value);
+          return false;
+        },
+        child: SingleChildScrollView(
+          child: SizedBox(height: height),
+        ),
+      );
+    }
+    await tester.pumpWidget(buildFrame(1200.0));
+    // Initial metrics notification.
```

Comment:
@Piinks maybe cancel the initial metrics event could resolve the tests, I am not sure.